go-controller/pkg/node/management-port.go

```diff
@@ -158,6 +147,23 @@ func managementPortReady() (bool, error) {
 	if !strings.Contains(stdout, "actions=output:"+ofport) {
 		return false, nil
 	}
+
+	// In the case of transitioning from random MAC to ip-mapped MAC, following code will get the
+	// MAC from mgmt port, and check if the address is applied in openflow table 35 or not
+	mac, _, err := util.RunOVSVsctl("--if-exists", "get", "interface", k8sMgmtIntfName, "mac")
+	if err != nil {
+		return false, nil
+	}
+	// check mac address in openflow
+	stdout, _, err = util.RunOVSOfctl("--no-stats", "--no-names", "dump-flows", "br-int",
+		"table=35,dl_dst="+mac)
+	if err != nil {
```

Comment:
NIT: would it be useful to add a log here? My thinking is that if k8sMgmtIntfName returned a mac, but table 35 is not yet programmed this is going to be a little difficult to know it happened, agree?

Reply:
make sense, fixed